src/main/java/core/basesyntax/dao/FruitDataReader.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.dao;
+
+import java.util.List;
+
+public interface FruitDataReader {
+    List<String> readTransactions();
+}
```

Comment:
here I would suggest using more abstract class and method name
in fact, you will read Strings, no matter if it is info about fruits, or something else